equinix-openapi-metal/api/openapi.yaml

```diff
@@ -18487,6 +18507,7 @@ components:
         line:
           enum:
           - baremetal
+          - reserved
```

Comment:
Rather than adding the `reserved` option to the enum, let's remove the enum entirely, so the `line` property becomes:

```
line:
  type: string
```

Reply:
updated.